### PR TITLE
Styling presets and qr code adjustment

### PR DESCRIPTION
**Pull request by benalleng**
After embedding the iframe into my personal website I was presented with a few pain points.

1. The qr codes overlapped or were partially covered by other parts of the website. In my case it was simply blank space but there could be some overlapping of other important information if it always extends out beyond the iframe border.
2. The paynym QR code was too tiny to be reliably scanned from an external device when looking from a phone -> desktop
3. Adjusting the color scheme of the entire iframe and its innerHTML was quite tedious and could be streamlined with presets.

To adjust this I created a translation effect on each of the qr codes to better keep them inside of the bounds of the iframe wherever it is embedded.  Furthermore the styling of the two pages presented me with an issue of manually setting the colors of each individual element.  A root class with presets has been added to better organize the colors for the sake of simplicity and possibly giving people an easier time of adjusting the color of the entire element more uniformly .

### CODE REVIEW

**Comment by benalleng** [comment]
Drawbacks specifically for the qr code adjustment are that when it is enlarged it covers the amount which could present problems if the payee mis-types or otherwise misreads the amount before scanning the qr code too hastily.  Perhaps there is a medium of how to best adjust the qr code to stay within the general bounds of the iframe while also not covering the important information.

File: config.py [hunk 1]
```diff
@@ -96,8 +96,9 @@ def check_set_node_conf(name, default, node_conf):
     payment_methods.append(method_config)
 
 supported_currencies = get_opt("supported_currencies", ["USD"])
-if "BTC" in supported_currencies:
-    supported_currencies.append("sats")
+if "BTC" and not "sats" in supported_currencies:
+    BTC_index = supported_currencies.index("BTC")
+    supported_currencies.insert(BTC_index, "sats")
```
Comment (kristapsk) [inline config.py:101]: Why this change? I personally prefer alphabetical order. Besides, seems to be out of scope of this styling change PR.

Comment (benalleng) [reply config.py]: Right sorry, didn't mean to have that commit on this pull request. I have removed it.  With that said I do not believe that the supported_currency list orders them alphabetically i believe it is just ordered how they are in the config.toml file and then sats is appended to the very end of that list.

Comment (kristapsk) [reply config.py]: Ahh, right, I always add them to `config.toml` in alphabetal order. And then "sats" actually go after USD, which is not alphabetical.

Comment (benalleng) [reply config.py]: I agree it is worth having some consistency in the order across instances of Satsale, and I do agree that alphabetical order is clear and consistent but I think there is some value in trying to keep the bitcoin options together, having sats as the default selection initially but then being both last on the list and so far away from BTC seems a bit confusing as a list of fiat options gets large

**Comment by nickfarrow** [comment]
Awesome PR man! The QR has been bugging me for ages 😅

**Comment by benalleng** [comment]
As a question for the future is there a way to parse flake8 tests for my code in the future? i just ran it on my code but seeing all the test failures is a bit overwhelming and nearly impossible to go through!! Is there a way to match my flake8 ignore to match with this repos linter? Not really familiar with running tests on my code :sweat_smile:

**Comment by kristapsk** [comment]
You need to have flake8 installed locally and then you run `./test/lint/lint-python.sh` manually.

**Comment by nickfarrow** [comment]
> As a question for the future is there a way to parse flake8 tests for my code in the future? i just ran it on my code but seeing all the test failures is a bit overwhelming and nearly impossible to go through!! Is there a way to match my flake8 ignore to match with this repos linter? Not really familiar with running tests on my code sweat_smile

That's a good point, might be worth running the linter test prior to the python version tests (so just one fails and doesn't run the others -> only one ❌)

And apologies the tests didn't run immediately, seems github has some "New contributor needs approval".. I just had to approve a 2nd time? Odd...

[Thanks for the great improvements!](https://twitter.com/SatoshiSale/status/1593979868219248640)
